Integrations/python/deephaven/__init__.py

```diff
@@ -136,6 +135,8 @@ def initialize():
 Figure = None  # variable for initialization
 PlottingConvenience = None  # variable for initialization
 
+if jpy.has_jvm():
+    DynamicTableWriter = jpy.get_type("io.deephaven.db.v2.utils.DynamicTableWriter")
```

Comment:
My concern is that this is not done in the same way as all of the other imported types.  Why is this being done differently than `PythonFunction`, `PythonListenerAdapter`, etc.?

Reply:
This isn't done the same as the other methods in the module because it's different than those listed.  DynamicTableWriter is only meant to be a convenience alias for a jpy.get_type() call to fetch the DynamicTableWriter object from io.deephaven.db.v2.utils.  It shouldn't have to be any more complex than this.

Reply:
What you are saying does not make sense.

1) The assertion that `DynamicTableWriter` is different is just not true.  Look at `PythonFunction` and `PythonListenerAdapter`.  Those are just imported types.
2) `PickedResult` and `Config` are not like what you did.  All of these methods use `__initializer__`, as can be seen here:
```
def initialize():
    __initializer__(jpy.get_type("io.deephaven.configuration.Configuration"), Config)
    __initializer__(jpy.get_type("io.deephaven.db.util.PickledResult"), PickledResult)
    __initializer__(jpy.get_type("io.deephaven.integrations.python.PythonListenerAdapter"), PythonListenerAdapter)
    __initializer__(jpy.get_type("io.deephaven.integrations.python.PythonFunction"), PythonFunction)
```

This is not the same as:
```
if jpy.has_jvm():
    DynamicTableWriter = jpy.get_type("io.deephaven.db.v2.utils.DynamicTableWriter")
```